app/javascript/packs/bikes_search/components/BikeSearch.js

```diff
@@ -4,6 +4,7 @@ import React, { Fragment, Component } from "react";
 
 import Loading from "../../Loading";
 import honeybadger from "../../utils/honeybadger";
+import TimeParser from "../../utils/time_parser";
 
```

Comment:
Pre-existing bug that now raises an exception since there's a stricter separation between webpack and sprockets-managed JS.